src/components/forms/AppInput.vue

```diff
@@ -80,6 +83,7 @@ const props = withDefaults(
     sameAs: undefined,
     pattern: undefined,
     prefix: undefined,
+    suffix: undefined,
```

Comment:
Why are the values here initialised with `undefined`?

Reply:
It seems strange to me too but if I don't set a default on optional props then I get an ESLint warning:

![image](https://github.com/beabee-communityrm/beabee-frontend/assets/2084823/69a710e3-08a3-4dff-b50a-23fdbce41a4f)

Reply:
okay